src/concurrently.ts

```diff
@@ -116,9 +122,13 @@ export function concurrently(
     const commandParsers: CommandParser[] = [
         new StripQuotes(),
         new ExpandNpmShortcut(),
-        new ExpandNpmWildcard()
+        new ExpandNpmWildcard(),
     ];
 
+    if (options.passthroughArguments) {
+        commandParsers.push(new ExpandArguments());
```

Comment:
Doesn't seem like there's much usefulness in passing `additionalArguments` to every command.
WDYT of passing them directly to `ExpandArguments` instance?

Could save having to publicly document and support it too.

Reply:
That sounds like a very good idea! I've updated the PR accordingly.